manifests/config.pp

```diff
@@ -133,4 +133,66 @@
       }
     }
   }
+
+  if $foreman_proxy::manage_certificates {
+    file { "${foreman_proxy::config_dir}/ssl_ca.pem":
+      ensure => file,
+      source => $foreman_proxy::ssl_ca,
+      owner  => $foreman_proxy::user,
```

Comment:
In general we make `root` the owner of these files so the proxy can't modify them. That makes sense to me because the installer manages them and we don't want 2 managers.